sdk/python/assets/assets-in-registry/share-data-using-registry.ipynb

```diff
@@ -440,27 +441,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# fetch the data from workspace\n",
-    "data_in_workspace = ml_client_workspace.data.get(\n",
-    "    name=\"transformed-nyc-taxt-data-ws\", version=version\n",
-    ")\n",
-    "print(\"data from workspace:\\n\\n\", data_in_workspace)\n",
-    "\n",
-    "# change the format such that the registry understands the data (when you print the data_ready_to_copy object, notice the asset id\n",
-    "data_ready_to_copy = ml_client_workspace.data._prepare_to_copy(\n",
-    "    data_in_workspace, name=\"transformed-nyc-taxt-data-shared-from-ws\", version=version\n",
-    ")\n",
-    "print(\"\\n\\ndata ready to copy:\\n\\n\", data_ready_to_copy)\n",
-    "# copy the data from registry to workspace\n",
-    "ml_client_registry.data.create_or_update(data_ready_to_copy).wait()"
+    "# Sharing data from registry to workspace\n",
```

Comment:
Should this be: share data from workspace to registry?

Reply:
Fixed